com.supermodel3.Supermodel.yaml

```diff
@@ -0,0 +1,38 @@
+id: com.supermodel3.Supermodel
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: supermodel
+separate-locales: false
+finish-args:
+  - --device=all
+  - --persist=.supermodel
+  - --filesystem=host:ro
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --allow=bluetooth
+modules:
+  - shared-modules/glu/glu-9.json
+
+  - name: supermodel
+    buildsystem: simple
+    build-commands:
+      - make
+          -f Makefiles/Makefile.UNIX
+          NET_BOARD=1
+      - install -Dm 0755 bin/supermodel ${FLATPAK_DEST}/bin/supermodel
+      - install -Dm 0644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - cp -r Assets ${FLATPAK_DEST}
+      - cp -r Config ${FLATPAK_DEST}
+    sources:
+      - type: git
+        url: https://github.com/trzy/Supermodel.git
+        branch: master
```

Comment:
```suggestion
```

NEVER a branch

Reply:
"command line only" you mean it doesn't display anything? I though this was an arcade machine emuator?

Reply:
Has no UI other than the command line